File: equipo/typescript/median_of_two_sorted_arrays.ts
function findMedianSortedArrays(nums1: number[], nums2: number[]): number {
    let nums = nums1.concat(nums2);
    let res = 0
    nums.sort((a,b) => a-b);

     if (nums.length%2 == 0){
        res = (nums.length/2)
        return ((nums[res-1]+nums[res])/2)
    }else{
        res = Number(((nums.length/2)+0.5))
        return nums[res-1]
    }
};

File: equipo/Java/Median_of_Two_Sorted_Arrays.java
class Solution {
    public double findMedianSortedArrays(int[] nums1, int[] nums2) {
        int [] nums = new int[nums1.length+nums2.length];
        System.arraycopy(nums1,0,nums,0,nums1.length);
        System.arraycopy(nums2,0,nums,nums1.length,nums2.length);
        
        Arrays.sort(nums);

        if(nums.length %2 == 0){
            double res = (nums.length/2);
            int result = (int) res;
            return ((double) (nums[result-1]+nums[result])/2);
        }else{
            double res = Math.ceil((double) nums.length/2);
            int result = (int) res;
            return nums[result-1];
        }

        
    }
}